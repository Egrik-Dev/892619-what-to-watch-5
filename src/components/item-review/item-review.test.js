import React from 'react';
import renderer from 'react-test-renderer';
import ItemReview from './item-review';

const comment = {
  id: 1,
  user: {
    id: 18,
    name: `Ben`
  },
  rating: 3.7,
  comment: `I love this movie. This film is a milestone in cinematography. Great Immersive camera-work.`,
  date: `2020-10-09T13:38:44.769Z`
};

it(`Should ItemReview render correctly`, () => {
  const tree = renderer
    .create(
        <ItemReview
          comment={comment}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
