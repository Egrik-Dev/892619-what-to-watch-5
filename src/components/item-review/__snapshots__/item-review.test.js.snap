// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ItemReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      I love this movie. This film is a milestone in cinematography. Great Immersive camera-work.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Ben
      </cite>
      <time
        className="review__date"
        dateTime="2020-10-09"
      >
        October 09, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    3.7
  </div>
</div>
`;
